front50-migrations/front50-migrations.gradle

```diff
@@ -17,6 +17,7 @@
 dependencies {
   implementation project(":front50-core")
 
```

Comment:
You need this for ?

Reply:
`com.netflix.spinnaker.fiat.model.resources.Permissions` comes from fiat-core.